packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -356,11 +356,7 @@ function commitLifeCycles(
       if (enableSuspense) {
         if ((finishedWork.mode & StrictMode) === NoEffect) {
           // In loose mode, a placeholder times out by scheduling a synchronous
-          // update in the commit phase. Use `updateQueue` field to signal that
-          // the Timeout needs to switch to the placeholder. We don't need an
-          // entire queue. Any non-null value works.
-          // $FlowFixMe - Intentionally using a value other than an UpdateQueue.
-          finishedWork.updateQueue = emptyObject;
```

Comment:
This should never have been landed.

Reply:
An explanation for why you don't like this hack leads to better software and smarter teammates!